rag_experiment_accelerator/ingest_data/acs_ingest.py

```diff
@@ -225,3 +204,38 @@ def do_we_need_multiple_questions(question, azure_oai_deployment_name):
     except ContentFilteredException as e:
         logger.error(e)
         return False
+
+
+def chunks_to_index_documents(chunks):
```

Comment:
Is this function taking a list of dictionaries and returning a list of dictionaries?
For most of the dictionary elements, nothing changes other than....

- id is generated from a hash
- `titleVector` becomes `title_vector`

Is this code actually needed?

Reply:
this function does more, it renames the keys, replaces None with empty strings and empty arrays, all this logic is required for changing chunks to documents ready for upload to azure search index. before this PR all the code was inside the main function, I've extracted it to be more readable.